src/Containers/Microsoft.NET.Build.Containers/Tasks/CreateImageIndex.cs

```diff
@@ -63,6 +23,8 @@ public void Dispose()
         _cancellationTokenSource.Dispose();
     }
 
+    private bool IsLocalPull => string.IsNullOrEmpty(BaseRegistry);
```

Comment:
nit: I think that `IsNullOrWhiteSpace` would be a better fit here. Just in case someone will provide only white spaces :)

Reply:
thanks, I haven't thought of that! done.